ports/renesas-ra/Makefile

```diff
@@ -108,7 +118,11 @@ INC += -I$(BOARD_DIR)/ra_cfg/fsp_cfg/bsp
 INC += -Idebug
 
 CFLAGS += -D$(CMSIS_MCU)
+ifeq ($(BOARD),$(filter $(BOARD),VK_RA4W1 VK_RA6M3 VK_RA6M5))
+CFLAGS += -DRA_HAL_H='<VK_$(CMSIS_MCU)_hal.h>'
```

Comment:
Can you use the existing naming convention for the series ? i.e `RA4W1_hal.h`, `RA6M3_hal.h` and `RA6M5_hal.h`. It seems the only difference between the existing config files and the new ones you're adding is the default SCI debug channel, which can be defined in `mpconfigboard.h`. Also note that  `DEBUG_TX_PIN` and `DEBUG_RX_PIN` are never used anywhere so any defaults there (the EK maybe ?) should be fine.

Reply:
Yes I tried to use the default one hals (`*_hal.h`) but the default pins of the EK boards are other and even I select the channel in the `mpconfigbord.h` it is still pointing to the wrong ones. If you have a beter mechanism for same channel to select other alternative pins I am listening ...

Reply:
> If you have a beter mechanism for same channel to select other alternative pins I am listening ...

Well like I said the `DEBUG_RX_PIN` and `DEBUG_TX_PIN` pins are not really used anyway (grep and you won't find those defines used anywhere) so I'm not sure why you want to override them, however if you need to override them for some reason it's possible, but for now I think we just need to override the default debug channel.

There's more than one way to do override `DEBUG_CH` but I think there's a file that exists solely for this purpose, which is `ra*_ek_conf.h` and for some reason it's not included anywhere, but we can use it. If you just rename `vk_ra6m1_conf.h` to `ra6m5_ek_conf.h` and include that file in `RA6M5_hal.h` then you have the debug channel defined (and of course every `RAx_hal.h` should include its `rax_ek_conf.h`). You can also use the same file to define `USE_DBG_PRINT` when needed.

Reply:
Yes I do it with a separate file because i was afraid it will break the compile of the existing boards and I hadn't noticed it is not used. OK if you insist to be in 1 file it will be, I will fix it.

Reply:
> OK if you insist to be in 1 file it will be, I will fix it.

Not insisting but IMHO I think it's much cleaner this way, specific board names should Not be in the Makefile I know there were some board names in the Makefile before, but this doesn't mean we can't clean it up. My goal here is to help you make these changes as generic as possible, this will hopefully speed up the merging of this PR and allow others to reuse your work here.

Reply:
Oh no please take your time, I'm currently testing your PR with these suggested changes, and reporting back anything that I think might be helpful ;)